python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -169,14 +176,25 @@ def metadata(self):
 
     def pin_memory_(self):
         """In-place operation to copy the feature to pinned memory."""
-        self._tensor = self._tensor.pin_memory()
+        self._is_inplace_pinned = set()
+        # https://github.com/pytorch/pytorch/issues/32167#issuecomment-753551842
```

Comment:
Add context instead just paste the link here.

# blababla, see more details:
# https://github.com/pytorch/pytorch/issues/32167#issuecomment-753551842

Reply:
Added context, can you check?